<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>randomMethod</class>
 <widget class="QWidget" name="randomMethod">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1600</width>
    <height>700</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <widget class="QPushButton" name="exitButton">
   <property name="geometry">
    <rect>
     <x>1500</x>
     <y>650</y>
     <width>91</width>
     <height>41</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>11</pointsize>
    </font>
   </property>
   <property name="text">
    <string>вернуться</string>
   </property>
  </widget>
  <widget class="QPushButton" name="nextImageButton">
   <property name="geometry">
    <rect>
     <x>1450</x>
     <y>520</y>
     <width>141</width>
     <height>51</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>16</pointsize>
    </font>
   </property>
   <property name="text">
    <string>далее</string>
   </property>
  </widget>
  <widget class="QPushButton" name="prevImageButton">
   <property name="geometry">
    <rect>
     <x>1450</x>
     <y>580</y>
     <width>141</width>
     <height>51</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>16</pointsize>
    </font>
   </property>
   <property name="text">
    <string>назад</string>
   </property>
  </widget>
  <widget class="QLabel" name="answer_image">
   <property name="geometry">
    <rect>
     <x>60</x>
     <y>300</y>
     <width>63</width>
     <height>20</height>
    </rect>
   </property>
   <property name="text">
    <string>TextLabel</string>
   </property>
  </widget>
  <widget class="QLabel" name="test_image">
   <property name="geometry">
    <rect>
     <x>60</x>
     <y>30</y>
     <width>63</width>
     <height>20</height>
    </rect>
   </property>
   <property name="text">
    <string>TextLabel</string>
   </property>
  </widget>
  <widget class="QWidget" name="histogramWidget" native="true">
   <property name="geometry">
    <rect>
     <x>280</x>
     <y>20</y>
     <width>600</width>
     <height>400</height>
    </rect>
   </property>
  </widget>
  <widget class="QWidget" name="histogramWidget_2" native="true">
   <property name="geometry">
    <rect>
     <x>910</x>
     <y>20</y>
     <width>600</width>
     <height>400</height>
    </rect>
   </property>
  </widget>
  <widget class="QLabel" name="label_6">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>270</y>
     <width>221</width>
     <height>31</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>12</pointsize>
    </font>
   </property>
   <property name="text">
    <string>Ответ метода рандома</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_5">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>0</y>
     <width>221</width>
     <height>31</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>12</pointsize>
    </font>
   </property>
   <property name="text">
    <string>Тестовое изображение</string>
   </property>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
